commit 80b43aaae27cfdf543483cd55d9012e37219350a
Author: Alex Richardson <arichardson.kde@gmail.com>
Date:   Wed Jan 29 21:56:27 2014 +0100

    Fix build with clang
    
    If we don't cast to unsigned int clang complains with following message:
    
    error: case value evaluates to 2690980318, which cannot be narrowed to
    type 'int'

--- a/dnssd/servicemodel.cpp
+++ b/dnssd/servicemodel.cpp
@@ -87,7 +87,7 @@ QVariant ServiceModel::data(const QModelIndex &index, int role) const
     if (!index.isValid()) return QVariant();
     if (!hasIndex(index.row(), index.column(), index.parent())) return QVariant();
     const QList<RemoteService::Ptr> srv=d->m_browser->services();
-    switch (role) {
+    switch ((uint)role) {
 	case Qt::DisplayRole: 
 	    switch (index.column()) {
 		case ServiceName: return srv[index.row()]->serviceName();
